documentation/src/main/asciidoc/introduction/Configuration.adoc

```diff
@@ -2,10 +2,11 @@
 == Configuration and bootstrap
 
 We would love to make this section short.
-Unfortunately, there's several distinct ways to configure and bootstrap Hibernate, and we're going to have to describe at least two of them in detail.
+Unfortunately, there are several distinct ways to configure and bootstrap Hibernate, and we're going to have to describe at least two of them in detail.
 
 The four basic ways to obtain an instance of Hibernate are shown in the following table:
 
+.Four ways to bootstrap Hibernate
```

Comment:
without this table title, the next table will have title of table 2, a little bit inconsistent.